packages/next/next-server/server/image-optimizer.ts

```diff
@@ -111,6 +111,14 @@ export async function imageOptimizer(
     return { finished: true }
   }
 
+  if (s && s !== 'true') {
```

Comment:
Consider using `1` instead since its slightly shorter and this API is not meant to be used directly by the user